app/models/organization_account_request.rb

```diff
@@ -0,0 +1,17 @@
+class OrganizationAccountRequest
+  include ActiveModel::Model
+  include ActiveModel::Validations::Callbacks
+  attr_accessor :name, :email, :requester_name, :phone_number, :city_town, :country, :province_state
+
+  before_validation :normalize_phone
+
+  validates :name, :email, :city_town, :country, :province_state, presence: true
+  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
+  validates :phone_number, presence: true, phone: {possible: true, allow_blank: true}
+
+  private
+
+  def normalize_phone
```

Comment:
Can't we use the `Phoneable` module instead? This method is already there.

`app/models/concerns/phoneable.rb`

Reply:
In phonebale, we have a `before_save` callback, and since we aren't saving these records into the database and only validating them, it doesn't work for me, hence i had to add the method separately 😬

Reply:
but I noticed now the Organization module has Phoneable included but still defines the method as well, I removed it from there.

Reply:
yes, but I cant use the `before_save` callback in my class 🤔

Reply:
Ooh I understand now. I wonder if we can conditionally call the before save callback in the module then. Ie it gets called unless the calling class name is this organization requests class? Don’t spend ages on it…but I think we can try that?

Reply:
OK no problem. Let's move on with your current!